roc/context.go

```diff
@@ -40,7 +40,10 @@ type Context struct {
 // Open a new context.
 // Allocates and initializes a new context. May start some background threads.
 // User is responsible to call Context.Close to free context resources.
-func OpenContext(config ContextConfig) (*Context, error) {
+func OpenContext(config ContextConfig) (ctx *Context, err error) {
+	logWrite(LogDebug, "entering roc_context_open(): config=%+v", config)
+	defer logWrite(LogDebug, "leaving roc_context_open(): context=%p err=%v", ctx, err)
```

Comment:
Since these log messages surround theb whole OpenContex, not just the call to native roc_open_context, I suggest to replace roc_open_context with OpenContext in the message.

Same for other methods. Since we're logging the whole go method, let's use go method name.

Reply:
done